docs/docs/self-host/env.mdx

```diff
@@ -196,6 +196,47 @@ With all the SSO providers listed below, replace the all caps values with values
 }
 ```
 
+<details>
+    <summary>See complete Okta instructions</summary>
+    <div>
+    <ol>
+    <li>
+<strong>Create an OIDC Web application:</strong>
+<img src="/images/guides/SSO-Okta-1.png" />
+</li>
+
+<li>
+<strong>Allow refresh tokens and specify callback URLs:</strong>
+<ul>
+    <li><strong>Sign-in redirect URIs</strong> - <code>https://app.growthbook.io/oauth/callback</code></li>
+    <li><strong>Sign-out redirect URIs</strong> (optional) - <code>https://app.growthbook.io</code></li>
```

Comment:
We should add a note that `https://app.growthbook.io` should be replaced with your API_HOST if self hosting.

Also, since these instructions are useful for both self-hosting and cloud, do we want to move these instructions to a separate docs page?  I'm fine doing this now and figuring out a better structure later.

Reply:
Ya, lets do this in a followup